Anytype/Sources/ApplicationLayer/AppSceneHandler/AppSceneHandlerModifier.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import SwiftUI
+
+private struct AppSceneHandlerModifier: ViewModifier {
```

Comment:
It looks like AppScene**Url**HandlerModifier to me

Reply:
fixed